cmd/helmExecute.go

```diff
@@ -68,9 +67,35 @@ func runHelmExecute(helmCommand string, helmExecutor kubernetes.HelmExecutor) er
 		if err := helmExecutor.RunHelmPackage(); err != nil {
 			return fmt.Errorf("failed to execute helm package: %v", err)
 		}
-	case "push":
-		if err := helmExecutor.RunHelmPush(); err != nil {
-			return fmt.Errorf("failed to execute helm push: %v", err)
+	case "publish":
+		if err := helmExecutor.RunHelmPublish(); err != nil {
+			return fmt.Errorf("failed to execute helm publish: %v", err)
+		}
+	default:
+		if err := runHelmExecuteDefault(config, helmExecutor); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func runHelmExecuteDefault(config helmExecuteOptions, helmExecutor kubernetes.HelmExecutor) error {
+	if config.LintFlag {
+		if err := helmExecutor.RunHelmLint(); err != nil {
+			return fmt.Errorf("failed to execute helm lint: %v", err)
+		}
+	}
+
+	if config.PackageFlag {
```

Comment:
do we need the flag? Shouldn't we run package in any case of the default build cycle?

Otherwise maybe use `package` so that it can be `--package` as a cli flag and `package: true` in the config.

My preference would likely be to always run package

Reply:
flag was removed